### ISSUE TITLE
little bug

### ISSUE THREAD

**Issue by lesar** [funkia/list#36]
I think you have exchange row 18 with 19 in:  
[make-proxies](https://github.com/funkia/list/blob/master/scripts/make-proxies.js)  
so there is a comma problem in package.json

I have a strange warning too  

> WARNING in ./node_modules/list/dist/es/index.js
> (Emitted value instead of an instance of Error) Cannot find source file '../../src/index.ts': Error: 
> Can't resolve '../../src/index.ts' in '<path>/functionalHyperscriptTodoList/node_modules/list/dist/es'

regards

**Comment by paldepind** [comment]
Thank you for reporting the issue @lesar. It seems that you are indeed correct that I've borked the JSON :sob:

I've fixed the comma problem and released it as 2.0.10. Please see if that also fixes the other issue? :crossed_fingers:

**Comment by lesar** [comment]
No the problem is still here.  
I have publish a first release [functional todo list](https://github.com/lesar/functionalHyperscriptTodoList)  
If you do the same stuff: 
npm install,  
npm run start  

and the see at `http://localhost:8080/index.html` you can see my little example.  

If you see the warning seams List is seaching my index.ts on `../../src/index.ts` but this file is on `../../../../src/index.ts`  

I hope that is not my stupid conf error  

regards

**Comment by paldepind** [comment]
Thanks for the steps for reproducing the issue. I was able to see the error you described.

I think I've found the issue. The files generated by TypeScript included source maps and these source maps where included in the npm package. The sources themselves, however, was not included. This caused the browser to look for sources that didn't exist.

List v2.0.11 should fix the issues. I've tried it in your repository and it seemed to do the trick 😄
